unittests/VMRuntime/InterpreterTest.cpp

```diff
@@ -161,7 +161,7 @@ TEST_F(InterpreterTest, SimpleSmokeTest) {
   StringID printID = 1;
   StringID resultID = 2;
 
-  const unsigned FRAME_SIZE = 16;
+  const int FRAME_SIZE = 16;
```

Comment:
When we compile this test we see compilation errors for the three lines:
```
  BFG->emitMov(FRAME_SIZE + StackFrameLayout::ThisArg, 3);
  BFG->emitLoadConstString(FRAME_SIZE + StackFrameLayout::FirstArg, resultID);
  BFG->emitMov(FRAME_SIZE + StackFrameLayout::FirstArg - 1, 2);
```
They look like this:
```
(...)\unittests\VMRuntime\InterpreterTest.cpp(174,27): error C4308: negative integral constant converted to unsigned type [(...)\build\unittests\VMRuntime\HermesVMRuntimeTests.vcxproj]
```

Since `StackFrameLayout::ThisArg` and others are negative values, I converted the `FRAME_SIZE` to a signed integer and then explicitly convert the result of signed integer sum to an unsigned result.